src/components/frame/LandingPage/ColonyCard.tsx

```diff
@@ -0,0 +1,107 @@
+import { Plus } from '@phosphor-icons/react';
+import clsx from 'clsx';
+import React from 'react';
+
+import LoadingSkeleton from '~common/LoadingSkeleton/LoadingSkeleton.tsx';
+import { formatText } from '~utils/intl.ts';
+import Button from '~v5/shared/Button/Button.tsx';
+
+export interface ColonyCardProps {
+  colonyName?: string;
```

Comment:
I think all of the conditional rendering we're doing with skeletons and handling the create new colony function is putting way too many things into this component.
I would recommend the following steps to make this more sustainable:
1. Create 2 new components, a `ColonyCard` and `CreateNewColonyCard`
2. Instead of having all the props optional because the app may be loading, I would recommend creating a file structure something like this:
- `ColonyCardHandler` is a component which gets the colony and the loading state, if the app is loading it renders `ColonyCardSkeleton`, if we have all the necessary data, we render `ColonyCard` with all the props sent in defined and not nullable
- then the same for `CreateColonyCard` (can this one even be impacted by anything loading?)

Reply:
@bassgeta Ok, I will add `ColonyCardHandler` when I start creating logic for Landing page